demos/gptj_causal_lm/python/requirements.txt

```diff
@@ -1,7 +1,7 @@
 accelerate==0.20.3
 numpy==1.23
 torch==1.13.1
-onnx==1.15.0
+onnx
```

Comment:
Don't we risk incompatibility in time? Upgrading to the current latest with a specific number would give us more control.